desktop.system/desktop.system.s

```diff
@@ -1231,9 +1204,7 @@ file_loop:
         bit     LCBANK2
         bit     LCBANK2
         lda     COPIED_TO_RAMCARD_FLAG
-        php
-        bit     ROMIN2
-        plp
+        sta     ROMIN2
```

Comment:
Apparently (my notes are poorer than I'd like), if you use write operations on these softswitches it can leave the bank in read-only mode. Which should be moot here since we're banking in ROM, but that's why the code tries to use read operations as documented.